src/js/data/names.js

```diff
@@ -3436,7 +3436,7 @@ let first = {
         ["A.C.", 5],
         ["A.J.", 63],
         ["A.W.", 64],
-        ["ATorri", 65],
```

Comment:
Probably better to just ask me to do it and then have me try, since it's hard to write code to run on data without having that data handy.